### PR TITLE
Fixes #15224: Reorganise system technique to simplify execuing a dingle directive

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15224

### CODE REVIEW

File: techniques/system/common/1.0/promises.st [hunk 1]
```diff
@@ -56,73 +56,65 @@ body common control
...
-        "monitoring_hook_post",
+bundle agent rudder_init {
+  methods:
+      "Abort if agent is disabled"     usebundle => check_disable_agent;
+      "Abort on some cfengine version" usebundle => rudder_check_agent_version;
+      "Abort on invalid UUID"          usebundle => check_uuid;
+      "Startup report log"             usebundle => startup;
+      "Initialize ncf"                 usebundle => initialize_ncf;
+      "Define environment variables"   usebundle => get_environment_variables;
+&if(INITIAL)&
```
Comment (amousset) [inline techniques/system/common/1.0/promises.st]: not

File: techniques/system/common/1.0/promises.st [hunk 2]
```diff
@@ -56,73 +56,65 @@ body common control
...
 
-      };
-
-    !policy_server::
-
-      # This bundle is duplicated to avoid having empty lists and cfnull in bundlesequence
-      "specific_bs" slist => { };
+bundle agent common_system_end {
+  methods:
+&if(INITIAL)&
```
Comment (amousset) [inline techniques/system/common/1.0/promises.st]: not

File: techniques/system/common/1.0/promises.st [hunk 3]
```diff
@@ -56,73 +56,65 @@ body common control
...
+      "Post hook for monitoring methods"  usebundle => monitoring_hook_post;
+&endif&
+      "End report log"                    usebundle => endExecution;
+      # keep this at real end to avoid breaking reporting if syslog still works
+      "Restart syslog if needed"          usebundle => restart_services;
+}
 
-    !enterprise_edition::
-      "capability_option"      string => "--capability-file /opt/rudder/etc/agent-capabilities";
```
Comment (amousset) [inline techniques/system/common/1.0/promises.st]: this one is probably still needed

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
merged with a different commit id 4613e9192e86f6bb9d61e7f57bfb1c9692801338

**Pull request closed without merging**